_includes/header.html

```diff
@@ -33,7 +33,7 @@
       <li class="nav-item dropdown">
         <a href="#" class="nav-link dropdown-toggle" id="dropdown03" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Join us</a>
         <ul class="dropdown-menu" aria-labelledby="dropdown03">
-          <li class="dropdown-item"> <a class="nav-link" href="/vacancies">Openings</a></li>
+          <li class="dropdown-item"> <a class="nav-link" href="/careers">Careers</a></li>
```

Comment:
We don't need to have this complex redirect infrastructure. Let's just rename it ignoring the seo stuff.

Reply:
yeah, I wasn't too sure about this myself... reverted